etna/lib/etna/client.rb

```diff
@@ -17,6 +17,29 @@ def initialize(host, token, routes_available: true, ignore_ssl: false)
 
     attr_reader :routes
 
+    def signed_route_path(route, params)
```

Comment:
Veeery minor, but would it be possible to make the expiry a variable with default here?
I can imagine us wanting to generate signed route paths and pass them off to a service or hook where the turn around could be a bit longer.  We could still restrict the expiry to a max, say, 5 minutes?  It's also not important to address in this PR, just a thought.

Reply:
Hmm, since the approval happens right away (the message will be signed just before the request is sent) it actually should not matter how long the request is, and even a 10s expiration time is probably too much.